src/App.jsx

```diff
@@ -1,104 +1,31 @@
 import './App.scss';
+import postsFromServer from './api/posts.json';
```

Comment:
The import statement for `postsFromServer` is redundant as it is already imported on line 2. Consider removing the commented-out import on line 6 to clean up the code.